frontend/src/app/dora/service/api.service.ts

```diff
@@ -14,12 +14,22 @@ export class ApiService {
     if (type === 'pipelineTool') {
       return this.fetchPipelineData(params);
     } else if (type === 'board') {
-      return this.verifyJiraBoard(params);
+      return this.verifyBoard(params);
     } else if (type === 'sourceControl') {
       return this.fetchCodeBaseRepos(params);
     }
   };
 
+  verifyBoard(params: any): any {
```

Comment:
Typing check is lost here.  Could we use a union type to the param? just like this
```
  verifyBoard(params: JiraBoardParam | LinearBoardParam) {
```
and also add more clear type to the return value.

Reply:
fixed it as u suggested.